systems/registry/users/pb/users.proto

```diff
@@ -67,7 +67,7 @@ message User {
     string name = 1 [(validator.field) = { string_not_empty: true,  length_gt: 1 }];
     string email = 2  [(validator.field) = { human_error:"must be an email format" regex: "^$|^[a-z0-9._%+\\-]+@[a-z0-9.\\-]+\\.[a-z]{2,4}$"}];
     string phone = 3  [(validator.field) = { human_error:"must be a phone number format", regex: "^$|^(?:(?:\\(?(?:00|\\+)([1-4]\\d\\d|[1-9]\\d?)\\)?)?[\\-\\.\\ \\\\\\/]?)?((?:\\(?\\d{1,}\\)?[\\-\\.\\ \\\\\\/]?){0,})(?:[\\-\\.\\ \\\\\\/]?(?:#|ext\\.?|extension|x)[\\-\\.\\ \\\\\\/]?(\\d+))?$"}];
-    string uuid = 4  [(validator.field) = {uuid_ver: 0}];
+    string uuid = 4  [(validator.field) = {uuid_ver: 4}];
```

Comment:
Can we have conventional naming for ID field .I think it is better to rename this to `userID` or `ID`.

Reply:
We have this here because we are already using an ID field.